app/views/providers/has_other_proceedings/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= page_template page_title: t('.page_title'), template: :basic do %>
+
+  <%= render partial: 'shared/error' if @error %>
+
+  <% if @legal_aid_application.proceeding_types.any? %>
+    <%= govuk_fieldset_header(size: 'm') { t('.existing', count: "#{pluralize(@legal_aid_application.proceeding_types.count, 'proceeding')}")} %>
+    <div class="govuk-summary-list">
+      <% @legal_aid_application.proceeding_types.order(:created_at).each do |type| %>
+        <dl class="govuk-summary-list__row" id="proceeding_type_<%= type.code %>">
+          <dt class="govuk-summary-list__value"><%= type.meaning %></dt>
+          <dd class="govuk-summary-list__actions">
+            <%= link_to_accessible(
+                  t('.remove'),
+                  providers_legal_aid_application_has_other_proceedings_path(@legal_aid_application, id: type.code),
+                  class: 'govuk-link change-link',
+                  method: :delete,
+                  suffix: type.meaning
+                ) %>
+          </dd>
+          </dt>
```

Comment:
This closing `</dt>` tag doesn't seem to have a corresponding opening `<dt>` tag. Can it be removed?

Reply:
good spot